harness/tests/experiment/fixtures/pytorch_onevar_model.py

```diff
@@ -229,6 +229,16 @@ def build_validation_data_loader(self) -> torch.utils.data.DataLoader:
             raise ValueError(f"unknown dataloader_type: {self.hparams['dataloader_type']}")
 
 
+class OneVarApexTrial(OneVarTrial):
```

Comment:
Why is this distinct from `OneVarApexAMPTrial`?

Reply:
I don't want the wacky data for this test as we had for the autocasting & scaling tests; I want to be able to expect the loss to decrease monotonically. The trial class names are not descript as they are right now.